tox.ini

```diff
@@ -27,3 +27,13 @@ deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/dev_requirements.txt
     -e.
+
+[testenv:integration-spark-databricks-sql-endpoint-connector]
+basepython = python3.8
+commands = /bin/bash -c '{envpython} -m pytest -v tests/specs/spark-databricks-sql-endpoint-connector.dbtspec'
+           /bin/bash -c '{envpython} -m pytest -v -m profile_databricks_sql_connector {posargs} -n4 tests/integration/*'
```

Comment:
The profile name should be `profile_databricks_sql_endpoint_connector`, and we need to enable the custom integration tests for the profile.